packages/core/test/unit/application.test.ts

```diff
@@ -147,3 +137,5 @@ class FakeServer extends Context implements Server {
     this.running = false;
   }
 }
+
+class FakerServer extends FakeServer {}
```

Comment:
`FakeServer` and `FakerServer` are difficult to distinguish. Can you rename this class to `AnotherServer` please?